client/docs/swagger-ui/swagger.yaml

```diff
@@ -42452,25 +42452,36 @@ paths:
                           - ACCESS_DELETE
                           - ACCESS_ADMIN
                           - ACCESS_TRANSFER
+                          - ACCESS_FORCE_TRANSFER
                         default: ACCESS_UNSPECIFIED
                         description: >-
                           Access defines the different types of permissions that
                           a marker supports granting to an address.
 
                            - ACCESS_UNSPECIFIED: ACCESS_UNSPECIFIED defines a no-op vote option.
-                           - ACCESS_MINT: ACCESS_MINT is the ability to increase the supply of a marker
+                           - ACCESS_MINT: ACCESS_MINT is the ability to increase the supply of a marker.
                            - ACCESS_BURN: ACCESS_BURN is the ability to decrease the supply of the marker using coin held by the marker.
-                           - ACCESS_DEPOSIT: ACCESS_DEPOSIT is the ability to set a marker reference to this marker in the metadata/scopes module
+                           - ACCESS_DEPOSIT: ACCESS_DEPOSIT is the ability to set a marker reference to this marker in the metadata/scopes module.
```

Comment:
this comment isn't completely correct ... this permission is literally required to deposit tokens in the marker's account... as well as the scope/metadata thing...

Reply:
Updated both deposit and withdraw comments.